keps/0000-kep-template.md

```diff
@@ -86,9 +83,24 @@ A table of contents is helpful for quickly jumping to sections of a KEP and for
 
 [Tools for generating]: https://github.com/ekalinin/github-markdown-toc
 
+**ACTION REQUIRED:** There must be an issue in [kubernetes/enhancements](https://github.com/kubernetes/enhancements/issues) referencing this KEP and targeting a release milestone **before [Enhancement Freeze](https://github.com/kubernetes/sig-release/tree/master/releases)
+of the targeted release**.
+
+## Release Signoff Checklist
+
+Check these off as they are completed for the Release Team to track. These checklist items _must_ be updated for the enhancement to be released.
+
+- [ ] k/enhancements issue in release milestone and linked to KEP (insert link here)
+- [ ] KEP approvers have set the KEP status to `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] Graduation criteria is in place
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
```

Comment:
It seems like user-facing documentation on kubernetes.io should be in this checklist